.github/workflows/postman-docker.yaml

```diff
@@ -65,6 +58,13 @@ jobs:
           cd docker-compose
           ./start_dse_68_dev_mode.sh -j $JSONTAG -t $SGTAG
 
+      - name: Install Postman CLI
```

Comment:
even with the changes made in this PR to run against docker compose v2 (and thus ensure the `--wait` flag is used), it is still not guaranteed that postman will succeed on its first few attempts to call the JSON API. For some reason even though the JSON API health check passes and docker compose command exits, it appears the JSON API port 8080 is not immediately available on the Docker host. This change moves the postman CLI installation to after docker compose is started to make sure the port is available. Admittedly a bit of a hack.